docs/index-custom-title-page.html

```diff
@@ -0,0 +1,222 @@
+<style>
```

Comment:
Before we merge this, let's chat about global styles. The end result of this page looks nice, visually, but we need to handle these styles in a way that maintains consistency. Good start here!

Reply:
Yes! I also just talked with @elastic/obs-docs about this and we're interested in doing something similar for the Observability docs so it might be good to move any custom CSS in this PR out of /kibana into /docs. 🙂